src/content/reference/react/createRef.md

```diff
@@ -40,17 +40,17 @@ class MyComponent extends Component {
   // ...
 ```
 
-[See more examples below.](#usage)
+[Lihat contoh lebih banyak di bawah.](#usage)
 
-#### Parameters {/*parameters*/}
+#### Parameter {/*parameters*/}
 
-`createRef` takes no parameters.
+`createRef` tidak memerlukan parameter.
 
-#### Returns {/*returns*/}
+#### Balikan {/*returns*/}
```

Comment:
```suggestion
#### Kembalian {/*returns*/}
```

Use "Kembalian" instead of "Balikan", as per [this discussion thread](https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5769320).

Reply:
Thank you for early feedback ..will revise it later...